### PR TITLE
Feat/temps de liaison

### PR DESCRIPTION
**Pull request by tristan-gueguen**
Impacts:
- fichiers pdf (2 version)
- fichier excel
- fichier c1b
- settings company

TODO: tests

### CODE REVIEW

File: app/helpers/pdf/work_days.py [hunk 1]
```diff
@@ -120,6 +122,17 @@ def _get_detail_columns(include_support=False, include_expenditures=False):
         ),
     ]
 
+    if include_transfers:
+        columns.append(
+            Column(
+                name="transfer",
```
Comment (sandrica89) [inline app/helpers/pdf/work_days.py:128]: Sometimes column name is `ActivityType.TRANSFER.value` and sometimes `"transfer"` directly. Can it cause any issue if updating one but not the other?

File: app/helpers/xls.py [hunk 2]
```diff
@@ -220,10 +221,21 @@ def get_columns_in_main_sheet(require_expenditures, require_mission_name):
                 13,
                 light_green_hex,
             ),
+            (
+                "Temps de liaison",
```
Comment (sandrica89) [inline app/helpers/xls.py:225]: In other columns we don't specify "Temps de", shouldn't we put only `"Liaison"` here to be consistent?

File: app/models/activity.py [hunk 3]
```diff
@@ -26,15 +26,21 @@ def activity_versions_at(activities, at_time):
...
     DRIVE = "drive"
     WORK = "work"
     SUPPORT = "support"
+    TRANSFER = "transfer"
     __description__ = """
 Enumération des valeurs suivantes.
 - "drive" : conduite du véhicule
 - "work" : travail sans déplacement du véhicule
 - "support" : accompagnement ou disponibilité
+- "transfer": temps de liaison
```
Comment (sandrica89) [inline app/models/activity.py:43]: Same ramrk as before, I would simply put
```suggestion
- "transfer": liaison d'un point à un autre
```

File: app/templates/filters.py [hunk 4]
```diff
@@ -66,6 +66,8 @@ def format_activity_type(activity_or_break_type):
         return "conduite"
     if activity_or_break_type == ActivityType.SUPPORT:
         return "accompagnement"
+    if activity_or_break_type == ActivityType.TRANSFER:
+        return "temps de liaison"
```
Comment (sandrica89) [inline app/templates/filters.py:70]: Same remark as before
```suggestion
        return "liaison"
```

File: app/helpers/pdf/work_days.py (hunk 1 above)
Comment (tristan-gueguen) [reply app/helpers/pdf/work_days.py]: I did a bit of cleaning to leverage more on ActivityTypes.xxx.value and same for expenditures

**Review by rtaieb — APPROVED** [review]

**Pull request merged**